### ISSUE TITLE
Remove artifact dependency for java components if Java isn't installed.

### ISSUE THREAD

**Issue by jkotalik** [aspnet/SignalR#3083]
Trying to build universe locally and can't due to an artifact dependency:
```
 signtool : error : Did not find modules\SignalR\artifacts\build\\signalr-0.1.0-preview3-t00715fc90.jar at modules\SignalR\artifacts\build\\signalr-0.1.0-preview3-t00715fc90.jar
```

**Comment by anurse** [comment]
@mikaelm12 can you do this? It should be fairly simple. The `ArtifactInfo` is added in `build/repo.targets`, we just have to make it conditional on the variable we use to indicate if Java is present.

**Comment by mikaelm12** [comment]
Did it last night

**Comment by anurse** [comment]
Yep, saw that later :).

**Comment by mikaelm12** [comment]
Fixed here: 813a9e68bc6248113d88377e37879b9e6a5aef67
